<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrcCalc</class>
 <widget class="QMainWindow" name="CrcCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CrcCalc</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/general/OCR-cyan.ico</normaloff>../res/general/OCR-cyan.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QLabel" name="lblMsg">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="QListWidget" name="lstCRCSelect">
      <property name="maximumSize">
       <size>
        <width>148</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="5">
     <widget class="QPlainTextEdit" name="txtMsg"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnLoadFile">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>载入文件</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2" colspan="5">
     <widget class="QPlainTextEdit" name="txtEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>toolTip</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnCalc">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnClear">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="5">
     <widget class="QTextBrowser" name="txtBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtEdit</tabstop>
  <tabstop>btnCalc</tabstop>
  <tabstop>btnClear</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstCRCSelect</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>btnCalc</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
